src/Components/CustomElements/src/js/WebpackBuild.targets

```diff
@@ -0,0 +1,38 @@
+<Project>
+
+  <PropertyGroup>
+    <WebpackOutputFile>wwwroot\BlazorCustomElements.js</WebpackOutputFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <WebpackBuildInputs Include="$(MSBuildThisFileDirectory)**" Exclude="$(MSBuildThisFileDirectory)node_modules\**" />
+    <WebpackBuildInputs Include="$(BaseIntermediateOutputPath)webpackconfig.txt" />
+    <UpToDateCheckInput Include="@(WebpackBuildInputs)" />
+  </ItemGroup>
+
+  <!-- If package-lock.json has changed, perform an NPM install -->
+  <Target Name="NpmInstall"
+          Inputs="$(MSBuildThisFileDirectory)package-lock.json"
+          Outputs="$(BaseIntermediateOutputPath)package-lock.json">
+    <Message Importance="high" Text="Running npm install..." />
+    <Exec Command="npm install" WorkingDirectory="$(MSBuildThisFileDirectory)" />
```

Comment:
Let's be consistent and use `yarn install` and `yarn run` (see eng/targets/Npm.Common.targets) until we change _everything_ over. At least keep src/Components/ consistent please.